examples/timeseries/public.py

```diff
@@ -17,30 +18,48 @@
 # You should have received a copy of the GNU General Public License
 # along with GWpy.  If not, see <http://www.gnu.org/licenses/>.
 
-"""Plotting public LIGO data
+"""Accessing and visualising public GW detector data
 
-I would like to study the gravitational wave strain time-series around the
-time of an interesting simulated signal during the last science run (S6).
-
-These data are public, so we can load them directly from the web.
+Data from the current generation gravitational wave detectors are published
+by |GWOSCl| and freely available to the public.
+In this example we demonstrate how to identify times of a published
+GW detection event, and to download and visualise detector data.
 """
 
 __author__ = "Duncan Macleod <duncan.macleod@ligo.org>"
-__currentmodule__ = 'gwpy.timeseries'
+__currentmodule__ = "gwpy.timeseries"
+
+# Firstly, we can use the |gwosc-mod| Python package to query for the
```

Comment:
I'm not familiar with `gwosc-mod`; did you mean `gwosc` here?

Reply:
As above, this is a substitution reference defined in references.rst

https://github.com/gwpy/gwpy/blob/f9ce215df631c17485a085284780c56e7aa879b2/docs/references.rst#L18-L19